DMCompiler/DM/DMObject.cs

```diff
@@ -29,8 +30,8 @@ class DMObject {
 
         public bool IsRoot => Path == DreamPath.Root;
 
-        public List<DMASTObjectVarOverride>? danglingOverrides = null; // Overrides waiting for the LateVarDef event to happen
-
+        // Statements waiting for LateVarDef event to happen
+        public Dictionary<string, List<DMASTStatement>> danglingStatementsByUndefinedNames = new();
```

Comment:
Rename this to `DanglingStatementsByUndefinedNames` to fit the naming convention used throughout the rest of the project.